deployment/docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       - PYTHONPATH=/usr/share/qgis/python:/usr/share/qgis/python/plugins:/usr/share/qgis/python/plugins/inasafe:/home/app
       - DISPLAY=:99
       - C_FORCE_ROOT=True
+      - TASK_ALWAYS_EAGER=False
+      - ON_TRAVIS=False
```

Comment:
default docker compose settings for TASK_ALWAYS_EAGER and ON_TRAVIS.
TASK_ALWAYS_EAGER=False means the task will use Celery broker in async.

You can override this value in docker-compose.override.yml